test/fixtures/external-entity-references.js

```diff
@@ -0,0 +1,323 @@
+/**
```

Comment:
Change filename to **resolvable-ext-references-melinda.js**

Reply:
fixed